larq/models.py

```diff
@@ -41,29 +51,27 @@ def _count_fp_weights(layer):
     return layer.count_params()
 
 
-def _bit_to_MB(bit_value):
-    bit_to_byte_ratio = 1.0 / 8.0
-    byte_to_mega_bytes_ratio = 1.0 / (1024 ** 2)
-    return bit_value * bit_to_byte_ratio * byte_to_mega_bytes_ratio
+def _bit_to_kB(bit_value):
+    return bit_value / 8 / 1024
 
```

Comment:
Hmmm, I'm not sure about the unit change. MB is definately more suited to the size of the whole model whereas kB is more suited to per layer. Maybe change the total size to MB and keep per layer kB

Reply:
Fair point. I changed the total memory to MB.